index.html

```diff
@@ -0,0 +1,207 @@
+          of work, including the continuing trend for remote working post
+          pandemic, and what that might mean for employers and employees.
+        </p>
+      </div>
+
+      <div class="main__bottom">
+        <h2>2022.7.10(Thus) ~ 20(FRI)</h2>
+        <p>@Great Russell Street, London WC1B 3DG, England</p>
+      </div>
+    </main>
+
+    <!-- ---------- Program Section --------- -->
+    <section class="program container">
+      <h2>Main Program</h2>
+      <div class="line"></div>
+
+      <div class="program_container">
+        <!-- program 1 -->
+        <div class="program_list">
+          <div class="program__icon">
+            <i class="fas fa-balance-scale"></i>
+          </div>
+
+          <div class="program__title">
+            <h3>Work-life balance</h3>
+          </div>
+
+          <div class="program__description">
+            <p>
+              All employees face the challenge of finding the right work-life
+              balance.
+            </p>
+          </div>
+        </div>
+
+        <!-- program 2 -->
+        <div class="program_list">
+          <div class="program__icon">
+            <i class="fas fa-toolbox"></i>
+          </div>
+
+          <div class="program__title">
+            <h3>Diversity gap</h3>
+          </div>
+
+          <div class="program__description">
+            <p>
+              Has Covid changed the status quo for women and/or ethnic
+              minorities in the workplace?
+            </p>
+          </div>
+        </div>
+
+        <!-- program 3 -->
+        <div class="program_list">
+          <div class="program__icon">
+            <i class="fas fa-users"></i>
+          </div>
+
+          <div class="program__title">
+            <h3>Social structure</h3>
+          </div>
+
+          <div class="program__description">
+            <p>
+              Remote working, at least in some form, is likely here to stay for
+              businesses across nearly every sector.
+            </p>
+          </div>
+        </div>
+
+        <!-- program 4 -->
+        <div class="program_list">
+          <div class="program__icon">
+            <i class="fas fa-recycle"></i>
+          </div>
+
+          <div class="program__title">
+            <h3>Recreation</h3>
+          </div>
+
+          <div class="program__description">
+            <p>
+              What will work look like if our offices become virtual? How can
+              tech step up to boost workplace social interactions?
+            </p>
+          </div>
+        </div>
+      </div>
+
+      <a class="link" href="#">SEE THE WHOLE PROGRAM</a>
+      <div class="btn btn--primary m-2">
+        <a href="#">Join CC Global summit 2022</a>
+      </div>
+    </section>
+
+    <!-- ---------- Speaker Section --------- -->
+    <section class="speaker_section container">
+      <h2>Feature Speaker</h2>
+      <div class="line"></div>
+
+      <div class="speaker_list"></div>
+    </section>

```

Comment:
- [ ] In mobile view, featured speakers section's text is wrapping, kindly consider decreasing the size of image and amount of text to make the content fit: 

Your design  | Original design
------------- | -------------
<img width="425" alt="Screenshot 2022-06-23 at 22 35 38" src="https://user-images.githubusercontent.com/77038610/175371417-7e42fbd1-eb88-4ea8-b7fb-07d2b272c133.png"> | <img width="442" alt="Screenshot 2022-06-23 at 22 34 36" src="https://user-images.githubusercontent.com/77038610/175371015-4bdf1136-7434-4e70-a967-97dba447ab06.png">
cc2a-ad90-4c22-b66e-e40873f048d9.png">

Reply:
It takes too much time to fix this problem. But i used float left for the image, I thought it better to wrap the text around the image while decrease the width of the screen.